dummydataServer/controller/userController.js

```diff
@@ -0,0 +1,208 @@
+let user = require("../models/user.js");
+let books = require("../models/book.js");
+let borrowed =require("../models/borrowed.js");
+let review =require("../models/review.js");
+let favorites =require("../models/favorites.js")
+ 
+module.exports = {
+//API to borrow a book
+    borrowBook (req, res) {
+        const borrowerId = parseInt(req.params.userId, 10);
+        const bookIden = parseInt(req.params.bookId, 10);
+        let userExist;
```

Comment:
Nice, but you should rename `bookExist` and  `userExist` to `bookDetails` and userDetails respectively.

Reply:
Thank you. I will do that